Hunting Queries/OfficeActivity/TeamsFilesUploaded.yaml

```diff
@@ -0,0 +1,31 @@
+id: 90e198a9-efb6-4719-ad89-81b8e93633a7
+name: Files uploaded to teams and access summary
+description: |
+  'Provides a summary of files uploaded to teams chats and extracts 
+  the users and IP addresses that have accessed them.'
+requiredDataConnectors:
+  - connectorId: Office365
+    dataTypes:
+      - OfficeActivity
+tactics:
+  - InitialAccess
+  - Exfiltration
+relevantTechniques:
+  - T1199
+  - T1102
+  - T1078
+query: |
+
+  OfficeActivity 
+  | where RecordType =~ "SharePointFileOperation" 
+  | where SourceRelativeUrl contains "Microsoft Teams Chat Files" 
+  | where Operation =~ "FileUploaded" 
+  | join kind= leftouter ( 
+     OfficeActivity 
+      | where RecordType =~ "SharePointFileOperation" 
```

Comment:
Consider excluding access attempts from "app@sharepoint" which is a SharePoint farms service account and will generate a lot of noise.

Reply:
This is now excluded